energy/energy_generation_consumption.py

```diff
@@ -4,76 +4,114 @@
 
 # COMMAND ----------
 
-# Source prevents programmatic data fetch: "https://www.energyinst.org/__data/assets/file/0015/1421601/2023-Country-Transition-Tracker-Consolidated-Format.csv"
-current_dir = Path().resolve()
-file_path = current_dir / '2023-Country-Transition-Tracker-Consolidated-Format.csv'
+# Source prevents programmatic data fetch: https://ember-climate.org/data/api/
+# Curretnly requesting ITS to whitelist this request
+
+file_path = "/Workspace/Users/ysuzuki2@worldbank.org/mega-indicators/energy/yearly_full_release_long_format.csv"
```

Comment:
Let's update this to point to github location instead, ie 'https://raw.githubusercontent.com/weilu/mega-indicators/main/energy/2023-Country-Transition-Tracker-Consolidated-Format.csv' The link won't work until the PR is merged.